packages/sitecore-jss-proxy/tsconfig.json

```diff
@@ -5,8 +5,9 @@
     "module": "commonjs",
     "outDir": "dist/cjs",
     "strictBindCallApply": false,
+    "skipLibCheck": true,
```

Comment:
Do we have some library that is failing now? cloudsdk 0.4?

Reply:
yes, it's cloudsdk

Reply:
I agree to add this option, but may be there is a problem in cloudsdk 0.4? what is the problem, did you report it (created a bug)?

Reply:
Good idea, I will add a bug.
Otherwise this option just brings proxy package in line with other sitecore-jss packages.